src/content_script.js

```diff
@@ -222,9 +228,15 @@ function addFacebookBadge (target, badgeClassUId, socialAction) {
       document.querySelector(".fbc-has-badge.js-fbc-prompt-active").classList.remove("js-fbc-prompt-active");
       e.target.parentElement.parentNode.parentNode.classList.remove("active");
     });
+  } else if (socialAction === "share") {
+    target.addEventListener("click", (e) => {
+      e.preventDefault();
+      return false;
```

Comment:
Should we add explicit `return` statement in the `socialAction === "login"` block? Should we have a default `return` outside of the `if` blocks?

Reply:
Removed return false for now.